StyleCop.Analyzers/StyleCop.Analyzers/OrderingRules/SA1202ElementsMustBeOrderedByAccess.cs

```diff
@@ -122,6 +122,14 @@ private static void HandleMemberList(SyntaxNodeAnalysisContext context, SyntaxLi
             {
                 var currentSyntaxKind = member.Kind();
                 currentSyntaxKind = currentSyntaxKind == SyntaxKind.EventFieldDeclaration ? SyntaxKind.EventDeclaration : currentSyntaxKind;
+
+                // if the syntaxkind of this member (e.g. SyntaxKind.IncompleteMember) will not
```

Comment:
:bulb: Consider changing `syntaxkind` to `SyntaxKind`, as the former triggers a spelling violation.